### PR TITLE
otto 0.2.0

### PR DESCRIPTION
**Pull request by nicolai86**

### CODE REVIEW

File: Library/Formula/otto.rb [hunk 1]
```diff
@@ -18,164 +18,61 @@ class Otto < Formula
...
+      :revision => "803f01ea27e23d998825ec085f0d153cac01c828"
   end
 
-  go_resource "gopkg.in/flosch/pongo2.v3" do
-    url "https://gopkg.in/flosch/pongo2.v3.git",
-      :revision => "5e81b817a0c48c1c57cdf1a9056cf76bdee02ca9"
+  go_resource "golang.org/x/tools" do
+    url "https://go.googlesource.com/tools.git",
+      :revision => "4ad533583d0194672e7d3bc6fb8b67c8e905d853"
   end
```
Comment (bfontaine) [inline Library/Formula/otto.rb]: Any reason not to included these resources in the array above?

Comment (nicolai86) [inline Library/Formula/otto.rb]: @bfontaine the `go.googlesource.com/tools` needs to be imported as `golang.org/x/tools`, which is similar to what is done for [terraform](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/terraform.rb#L105)

Comment (nicolai86) [inline Library/Formula/otto.rb]: however, the `gopkg.in/flosch/pongo2.v3` resource can be included in `otto_deps`

Comment (bfontaine) [inline Library/Formula/otto.rb]: Thanks!

**Comment by MichielDeMey** [comment]
Totally waiting for this one. :+1:

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**